confidant/routes/services.py

```diff
@@ -579,12 +579,16 @@ def map_service_credentials(id):
     credentials = data.get('credentials', [])
     blind_credentials = data.get('blind_credentials', [])
     combined_credentials = credentials + blind_credentials
+    new_credentials = set(combined_credentials).difference(
+        _service.credentials + _service.blind_credentials
```

Comment:
Does `_service.credentials` return a list of ids?  Making sure it does not return an object / dictionary of some sort.

Reply:
good catch, it's actually a set so just change to `union()`